cmd/pow/cmd/prepare/prepare.go

```diff
@@ -248,64 +266,99 @@ The piece-size and piece-cid are printed to stderr. For scripting usage, its rec
 		if errCommP != nil {
 			c.Fatal(fmt.Errorf("calculating piece-size and PieceCID: %s", err))
 		}
+
+		if aggregate {
+			rootNd, err := dagService.Get(ctx, payloadCid)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get root node: %s", err))
+			}
+			manifestLnk := rootNd.Links()[0]
+			manifestNd, err := dagService.Get(ctx, manifestLnk.Cid)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get manifest node: %s", err))
+			}
+			manifestF, err := unixfile.NewUnixfsFile(context.Background(), dagService, manifestNd)
+			if err != nil {
+				c.Fatal(fmt.Errorf("get unifxfs manifest file: %s", err))
+			}
+			manifest := manifestF.(files.File)
+			fManifest, err := os.Create(args[1] + ".manifest")
```

Comment:
Is it possible that we generate the manifest file on the fly in `dagify` after walked the directory, without writing anything to the local directory?

Reply:
Actually, I'm writing it to the local directory as a feature. 
The manifest file is generated on the fly by the aggregator library and included in the batch UnixFS DAG.

Also, the `dagify` is per file and the manifest is a _batch scoped_ concept.

Reply:
Ah gotcha. I confused the order ;)